### PR TITLE
Docs: Fix formatting and clarity for SSO Setup

### PR DESCRIPTION
**Pull request by jjrcluedin**
## Description
Customer has pointed out a few bits that aren't quite right.
I have further fixed formatting (I think - unsure how to test locally properly) and just made some fields a bit more obvious as to what is necessary

### CODE REVIEW

**Comment by jjrcluedin** [comment]
@SolomiiaSnihur I'm not 100% sure how the formatting looks on this one. I think it's correct but would like your input :)

File: docs/020-deployment/howtos/020-setup-sso.md [hunk 1]
```diff
@@ -72,22 +72,41 @@ For more information about the client secret, see [Microsoft documentation](http
...
+
+    1. In the **Implicit grant and hybrid flows** section, select the **ID tokens** checkbox, and then select **Configure**.
+
+        ![Add_redirect_URIs_ID_tokens.png](../../assets/images/ama/howtos/configure-sso-8.png)
+
+1. In the **Platform configurations** section, in **Web**, add additional URIs to the existing one:
+
+    ![Add_redirect_URIs_additional.png](../../assets/images/ama/howtos/configure-sso-9.png)  
+    `https://departmentX.yourdomain.com`  
+    `https://app.yourdomain.com/auth/signin-oidc`
```
Comment (SolomiiaSnihur) [inline docs/020-deployment/howtos/020-setup-sso.md:107]:
```suggestion
1. In the **Platform configurations** section, in **Web**, add additional URIs to the existing one:

    `https://departmentX.yourdomain.com`
     
    `https://app.yourdomain.com/auth/signin-oidc`

    ![Add_redirect_URIs_additional.png](../../assets/images/ama/howtos/configure-sso-9.png)  
```

File: docs/020-deployment/howtos/020-setup-sso.md [hunk 2]
```diff
@@ -72,22 +72,41 @@ For more information about the client secret, see [Microsoft documentation](http
...
+
+    1. In **Front channel logout URI**, add a logout URL for your application.
+
+        ![Add_redirect_URIs_Logout_URL.png](../../assets/images/ama/howtos/configure-sso-7.png)  
+        `https://app.yourdomain.com/logout`
+
+    1. In the **Implicit grant and hybrid flows** section, select the **ID tokens** checkbox, and then select **Configure**.
+
+        ![Add_redirect_URIs_ID_tokens.png](../../assets/images/ama/howtos/configure-sso-8.png)
+
```
Comment (SolomiiaSnihur) [inline docs/020-deployment/howtos/020-setup-sso.md:102]:
```suggestion
    1. In **Redirect URIs**, add a redirect URI for your application.

        `https://app.yourdomain.com`
        
        ![Add_redirect_URIs_Redirect_URIs.png](../../assets/images/ama/howtos/configure-sso-6.png)

    1. In **Front channel logout URI**, add a logout URL for your application.

        `https://app.yourdomain.com/logout`
        
        ![Add_redirect_URIs_Logout_URL.png](../../assets/images/ama/howtos/configure-sso-7.png)  

    1. In the **Implicit grant and hybrid flows** section, select the **ID tokens** checkbox, and then select **Configure**.

        ![Add_redirect_URIs_ID_tokens.png](../../assets/images/ama/howtos/configure-sso-8.png)

```

File: docs/020-deployment/howtos/020-setup-sso.md [hunk 3]
```diff
@@ -72,22 +72,41 @@ For more information about the client secret, see [Microsoft documentation](http
 
 Redirect URI is the location to which the Microsoft identity platform redirects the user's client and sends security tokens after authentication.
 
+In the setup below, we will be working with fictitious domain 'yourdomain.com' and the two subdomains 'app' and 'departmentX'. 
+
+**departmentX** references the main CluedIn UI you would access.  
+And **app** references the backend application. By default, it is just app, but can be changed in the values file.
+
```
Comment (SolomiiaSnihur) [inline docs/020-deployment/howtos/020-setup-sso.md:79]:
```suggestion
In the setup below, we will be working with fictitious domain 'yourdomain.com' and the two subdomains:

- 'app' – references the main CluedIn UI you would access.

 - 'departmentX' – references the back-end application. By default, it is just app, but can be changed in the values file.

```

**Review by SolomiiaSnihur — CHANGES_REQUESTED** [review]
Reviewed changes. Added a bulleted list for subdomains. Moved user input before screenshots.

File: docs/020-deployment/howtos/020-setup-sso.md (hunk 3 above)
Comment (jjrcluedin) [reply docs/020-deployment/howtos/020-setup-sso.md:79]: I have amended this as there was a leading space, and i think bullet points are find to be together

Comment (jjrcluedin) [reply docs/020-deployment/howtos/020-setup-sso.md:79]: @SolomiiaSnihur are you happy with amendment above?

**Pull request merged**